packages/content-management/table_list_view_table/src/table_list_view_table.tsx

```diff
@@ -1075,10 +1073,10 @@ function TableListViewTableComp<T extends UserContentCommonSchema>({
       });
     };
 
-    updateQueryFromURL(urlState.s);
-    updateSortFromURL(urlState.sort);
-    updateFilterFromURL(urlState.filter);
-  }, [urlState, buildQueryFromText, urlStateEnabled]);
+    updateQueryFromURL(initialUrlState.s);
+    updateSortFromURL(initialUrlState.sort);
```

Comment:
Now that I re-read the code, I am surprise we call sometimes twice dispatch `onTableChange`. But I guess that's a tiny optimisation

Reply:
it is a good optimizaiton now, but would be automatically batched with react@18